components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/ExceptionCodes.java

```diff
@@ -317,7 +317,7 @@
     KEY_MANAGER_NOT_FOUND(901403, "Key Manager not found", 400, "Key Manager not found"),
     KEY_MANAGER_NAME_EMPTY(901404,
             "Key Manager name cannot be empty", 400,"Key Manager name cannot be empty"),
-    KEY_MANAGER_NOT_SUPPORT_OAUTH_APP_CREATION(901404, "Key Manager doesn't support Oauth application generation", 400,
+    KEY_MANAGER_NOT_SUPPORT_OAUTH_APP_CREATION(901405, "Key Manager doesn't support to Generate Client Application", 400,
```

Comment:
didn't we change this to "Key Manager doesn't support generating OAuth applications"?